modules/epomDspBidAdapter.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @name epomDspBidAdapter
+ * @version 1.0.0
+ * @description Adapter for Epom DSP and AdExchange
+ * @module modules/epomDspBidAdapter
+ */
+
+import { logError, logWarn, registerBidder } from '../src/adapters/bidderFactory.js';
+import { config } from '../src/config.js';
+const BIDDER_CODE = 'epomDsp';
```

Comment:
please update your bidder code: `Lower case alphanumeric with the only special character allowed is underscore.`